### PR TITLE
The hamburger menu is now changing color to black when intersecting section

### PR DESCRIPTION
**Pull request by hxmn**
The hamburger menu is now changing color to black when intersecting 'what' section

### CODE REVIEW

File: src/contexts/intersect-context.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React, { createContext, useState, useMemo } from "react";
+
+export const IntersectContext = createContext({});
+
+export const DokumentConsumer = IntersectContext.Consumer;
+
```
Comment (arneson) [inline src/contexts/intersect-context.js:6]: Sappa-skadad? ;)

Comment (hxmn) [reply src/contexts/intersect-context.js:6]: Haha ops! Copy paste!!!!!!!

**Comment by SimonTakman** [comment]
Should it be black or should we use the red or blue color?

**Comment by hxmn** [comment]
> Should it be black or should we use the red or blue color?

Tried them out and black looked the best IMO.

File: src/contexts/intersect-context.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+export const IntersectContext = createContext({});
+
+export const IntersectConsumer = IntersectContext.Consumer;
+
+export const IntersectProvider = ({ children }) => {
+  const [intersect, setIntersect] = useState();
+
+  const handleIntersect = entry => {
+    if (
+      entry.boundingClientRect?.top < 0 &&
```
Comment (arneson) [inline src/contexts/intersect-context.js:12]: Optional chaining browser support still excludes some browsers, so maybe avoid for now?

Comment (hxmn) [reply src/contexts/intersect-context.js:12]: OK!

**Pull request merged**